third_party/blink/renderer/core/inspector/inspector_dom_debugger_agent.h

```diff
@@ -195,6 +195,14 @@ class CORE_EXPORT InspectorDOMDebuggerAgent final
   InspectorAgentState::BooleanMap xhr_breakpoints_;
   InspectorAgentState::BooleanMap event_listener_breakpoints_;
   InspectorAgentState::BooleanMap csp_violation_breakpoints_;
+
+  // [replay]
+  static void ReplayNotifyWill(const String& eventName,
+                               EventTarget* eventTarget = nullptr,
+                               bool isCallback = false);
+  static void ReplayNotifyDid(const String& eventName,
+                              EventTarget* eventTarget = nullptr,
+                              bool isCallback = false);
```

Comment:
These should have comments in the declaration or definition describing what they do.

Reply:
Fixed. Also moved function declaration (now function prototype) out of class and to [here](https://github.com/replayio/chromium/pull/328/files#diff-9e73fbbf5fe06ab42947dc808f581af202959979facf65b44147dddd4b522b2fR730).